testfixtures/custom_type.go

```diff
@@ -5,25 +5,28 @@ import (
 	"fmt"
 )
 
-type CustomType []byte
+type CustomType [16]byte
```

Comment:
could you explain this fixture change?

Reply:
Yep: I wanted this to no longer be a natively supported SQL type